Dnn.AdminExperience/ClientSide/Users.Web/src/_exportables/src/components/UserTable/index.jsx

```diff
@@ -191,4 +191,4 @@ function mapStateToProps(state) {
     };
 }
 
-export default connect(mapStateToProps, null, null, { withRef: true })(UserTable);
\ No newline at end of file
+export default connect(mapStateToProps, null, null, { withRef: true, forwardRef: true, })(UserTable);
```

Comment:
The [6.0.0 release notes](https://github.com/reduxjs/react-redux/releases/tag/v6.0.0) say that `withRef` is replaced by `forwardRef`, do we still need `withRef` here?
```suggestion
export default connect(mapStateToProps, null, null, { forwardRef: true, })(UserTable);
```

Reply:
Oh, nice catch, it is in two places though, I'll send another commit soonish...